tests/components/ct-react-vite/tests/children.spec.tsx

```diff
@@ -49,8 +49,9 @@ test('render string as child', async ({ mount }) => {
 });
 
 test('render array as child', async ({ mount }) => {
-  const component = await mount(<DefaultChildren>{[4,2]}</DefaultChildren>);
-  await expect(component).toContainText('42');
+  const component = await mount(<DefaultChildren>{[<h4>{[4]}</h4>,<p>2</p>]}</DefaultChildren>);
```

Comment:
Let's include array of arrays, since these also seem possible in React.

```suggestion
  const component = await mount(<DefaultChildren>{[<h4>{[4]}</h4>,[[<p>[2,3]</p>]]]}</DefaultChildren>);
```